composer.json

```diff
@@ -5,10 +5,10 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.1",
-    "spryker-sdk/spryk": "^0.1.0 || ^0.2.0",
+    "spryker-sdk/spryk": "dev-improvment/te-2031-service-spryks as 0.2.1",
```

Comment:
Dont forget to reset this to the new version (including constraint update) before releasing!